MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -285,8 +296,24 @@ public void setStreamType(int type) {
         streamType = type;
     }
 
+    protected void dispatchMissingAdBreakStartEvent() {
+	    if (missingAdBreakStartEvent) {
+            dispatch(new AdBreakStartEvent(null));
+            dispatch(new AdPlayEvent(null));
+            missingAdBreakStartEvent = false;
+        }
+    }
+
     @Override
     public void dispatch(IEvent event) {
+	    if (!player.get().getPlayWhenReady() && (player.get().getCurrentPosition() == 0)) {
```

Comment:
This feels odd, and kind of backwards. Rather than have MuxBaseExoPlayer know when it needs to suppress events, and just sending those events all the time (and sometimes more times than others) from the IMA listener, I'd propose to flip this around, so that the IMA listener only emits the events when they _should_ be omitted.

Reply:
Moved from BaseExoPlayer to ImaSDKListener